v2/config_env_test.go

```diff
@@ -49,7 +49,7 @@ func TestEnvVariablesWithProfile(t *testing.T) {
 	defer os.RemoveAll(configFolderPath)
 
 	content := fmt.Sprintf(`{
-		"SomeProfile": {
+		"Default": {
```

Comment:
This test is meant to use a specific profile through `OSC_PROFILE` (see os.Setenv)